readme.md

```diff
@@ -12,3 +12,25 @@ A fun side project by some lovely folks at [Tighten Co.](http://tighten.co/).
 
 * [Api documentation](doc/api.md)
 * Chat about it on freenode IRC channel `#symposium`
+
+### Requirements
+
+* PHP >= 5.5.9
+* PHP [mcrypt extension](http://php.net/manual/en/book.mcrypt.php)
+* A [supported relational database](http://laravel.com/docs/5.1/database#introduction) and corresponding PHP extension
+* [Composer](https://getcomposer.org/download/)
+
+### Installation
+
+1. [Fork this repository](https://help.github.com/articles/fork-a-repo/)
+2. [Install dependencies](https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies)
+3. [Run database migrations](http://laravel.com/docs/5.1/migrations#running-migrations) against the desired database using the appropriate database connection name from [configuration](https://github.com/tightenco/symposium/blob/master/config/database.php)
+
+    ```
+    php artisan migrate --database=sqlite
```

Comment:
We normally would just set the info in `.env`, and then run it like this:

```bash
php artisan migrate --env=local
```

Also, since we have seeds, it would be:

```bash
php artisan migrate --env=local --seeds
```